fabric/src/main/java/dev/architectury/networking/fabric/SpawnEntityPacket.java

```diff
@@ -103,7 +105,7 @@ public static void receive(FriendlyByteBuf buf, NetworkManager.PacketContext con
                 entity.setYHeadRot(yHeadRot);
                 entity.setYBodyRot(yHeadRot);
                 if (entity instanceof EntitySpawnExtension ext) {
-                    ext.loadAdditionalSpawnData(buf);
+                    ext.loadAdditionalSpawnData(extBuf);
```

Comment:
Might need to close the new buffer here then?

Reply:
Good point, done in [3a687df](https://github.com/architectury/architectury-api/pull/220/commits/3a687dffd469db49ed5f39598cdcba40ef02e9b7)